ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -4402,6 +4402,7 @@ void HqlCppTranslator::buildSimpleExpr(BuildCtx & ctx, IHqlExpression * expr, CH
                 break;
             }
         }
+        break;
```

Comment:
Note: This was spotted by coverity at the same time.  It has no effect, but is prevents a potential bug if another case is added later.